tests/core/test_config.py

```diff
@@ -223,51 +222,35 @@ def check_memoization(p):
         assert p.max_history == 5
         assert p.priority == 3
 
-    def check_fallback(p):
-        assert p.fallback_action_name == "action_default_fallback"
-        assert p.nlu_threshold == 0.7
-        assert p.core_threshold == 0.7
-        assert p.priority == 2
-
-    def check_form(p):
+    def check_rule(p):
         assert p.priority == 1
 
     ensemble_dict = {
         "policies": [
             {"max_history": 5, "priority": 3, "name": "MemoizationPolicy"},
-            {
-                "core_threshold": 0.7,
-                "priority": 2,
-                "name": "FallbackPolicy",
-                "nlu_threshold": 0.7,
-                "fallback_action_name": "action_default_fallback",
-            },
-            {"name": "FormPolicy", "priority": 1},
+            {"name": "RulePolicy", "priority": 1},
         ]
     }
     ensemble = PolicyEnsemble.from_dict(ensemble_dict)
 
     # Check if all policies are present
-    assert len(ensemble) == 3
-    # MemoizationPolicy is parent of FormPolicy
+    assert len(ensemble) == 2
+    # MemoizationPolicy is parent of RulePolicy
     assert any(
         [
-            isinstance(p, MemoizationPolicy) and not isinstance(p, FormPolicy)
+            isinstance(p, MemoizationPolicy) and not isinstance(p, RulePolicy)
             for p in ensemble
         ]
     )
-    assert any([isinstance(p, FallbackPolicy) for p in ensemble])
-    assert any([isinstance(p, FormPolicy) for p in ensemble])
+    assert any([isinstance(p, RulePolicy) for p in ensemble])
 
     # Verify policy configurations
     for policy in ensemble:
         if isinstance(policy, MemoizationPolicy):
-            if isinstance(policy, FormPolicy):
-                check_form(policy)
+            if isinstance(policy, RulePolicy):
+                check_rule(policy)
             else:
                 check_memoization(policy)
-        elif isinstance(policy, FallbackPolicy):
-            check_fallback(policy)
 
 
 TEST_MIGRATED_MAPPING_POLICIES = [
```

Comment:
Actually, does it make sense to also remove this test (and all the stuff around the associated `rasa.core.config.migrate_mapping_policy_to_rules`? Assuming that Rasa 3.x won't know `MappingPolicy`, this migration code shouldn't work anyway (I think) 🤔  I guess this is another topic to discuss and make a decision on: Whether to leave the 1.x->2.x migration code in 3.x at all (I think not, as it might require some bits of code that we want to remove in 3.x, such as the implementation of the removed policies).

Reply:
So I looked at the code and I believe the migration code doesn't use the deprecated policies it just converts markdown to yaml so it wouldn't break anything in theory to keep it around (I think). I can also start that discussion on this.

Reply:
@wochinge says the migration code should go, but I am going to create a separate issue for removing it because it requires removing things from the cli as well 🍝 🗼 .

Reply:
Makes sense 🙂